src/guide/jbs-jdk-bug-system.md

```diff
@@ -72,7 +72,51 @@ A few things to keep in mind when filing an issue:
+
+The [Affects Version]{.jbs-field} field is used to indicate which releases an issue is applicable to, and to avoid having to set it to an exhaustive list of impacted releases the following assumptions are used to give that range:
+
+1) If a issue applicable to a feature release N it is assumed to be applicable to all (more recent) releases unless indicated otherwise (see (Rel)-na below).
+  - Note that if it is reported against an update release then all we can say is that it applicable to all the following update releases, not necessarily the next feature release as it may have been introduced in an update.  Given this, it is always important to try and reproduce the issue in either the corresponding feature release or the following.
+
+1) If an issue is applicable to release N, then it can't be assumed that it is applicable to older releases less than N. It may be, but in general this is less important to know this as the majority of issues are only fixed in the latest feature release.  If the issue is a crash or important in another way, then it becomes worthwhile to take the time to determine if it is relevant to earlier LTS releases.
+
+Another aspect of an issue is when, the feature it is a part of, was added or removed from the JDK, which in either case limits the range of releases the issue impacts.  Knowing that a feature was removed before the oldest currently supported release means it can be resolved as will-not-fix.,
+
+##### Setting the Affects Versions field
+Set the [Affects Versions]{.jbs-field} field to the lowest release value that the bug is known to be reproducible on.
+
+* The [Affects Version/s]{.jbs-field} isn't meant to be an exhaustive list of releases the issue is relevant to - it should initially be set to the release the issue was reproduced or identified on, and by implication it will be relevant on all releases past that point (see the [Usage of (Rel)[-na]{.jbs-label} Label](#usage-of-rel-na-label)). If it's later found to be applicable to an earlier release family then adding that earlier release is encouraged if the issue needs to be fixed in that release.
+* Do not add additional release values to [Affects Version/s]{.jbs-field} for the same release family. E.g. If there is the value [11.0.2]{.jbs-value}, don't add [11.0.5]{.jbs-value}, [11.0.7]{.jbs-value} etc. Adding an additional value for a separate release family where it's still reproducible, e.g. JDK [21]{.jbs-value}, is encouraged, especially if there is not currently a feature release value set, or, it has been a few releases since it was last reproduced/reviewed. E.g. [Affects Version/s]{.jbs-field} is JDK [8]{.jbs-value} but it is still relevant to the latest mainline release.
```

Comment:
```suggestion
* Do not add additional release values to [Affects Version/s]{.jbs-field} for the same release family. E.g. if there is the value [11.0.2]{.jbs-value}, don't add [11.0.5]{.jbs-value}, [11.0.7]{.jbs-value} etc. Adding an additional value for a separate release family where it's still reproducible, e.g. JDK [21]{.jbs-value}, is encouraged, especially if there is not currently a feature release value set, or, it has been a few releases since it was last reproduced/reviewed. E.g. [Affects Version/s]{.jbs-field} is JDK [8]{.jbs-value} but it is still relevant to the latest mainline release.
```

Reply:
I believe that technically that "E.g." is the start of the sentence - I may replace it with "For example, " as that may be cleaner

Reply:
Lower case, 'E.g. if' looks good to me.